pull-requests.md

```diff
@@ -20,11 +24,15 @@ Lastly, please follow the [WordPress Coding Standards](http://make.wordpress.org
 
 If you haven't submitted a pull request before, you'll want to install WP-CLI for local development:
 
-1. Clone this git repository on your local machine.
+1. Clone the WP-CLI git repository to your local machine: `git clone git@github.com:wp-cli/wp-cli.git ~/wp-cli`
```

Comment:
Would it be better here to use the "https://github.com" version of the request? It works for everyone, whereas for instance for me `git clone git@...` doesn't.